pkg/sidecarexec/os_config_test.go

```diff
@@ -13,9 +13,9 @@ import (
 )
 
 func Test_TrustedCertsCreateConfig(t *testing.T) {
-	backup, certs, close, err := createCertTempFiles(t)
+	backup, certs, closeFunc, err := createCertTempFiles(t)
 	assert.NoError(t, err)
-	defer close()
+	defer closeFunc()
```

Comment:
Golang lint v1.52.2 won't allow redefining builtin functions was giving this error - `redefines-builtin-id: redefinition of the built-in function close (revive)`